doc/lua_api.txt

```diff
@@ -1312,10 +1298,11 @@ Item types
 ----------
 There are three kinds of items: nodes, tools and craftitems.
 
-* Node (`register_node`): A node from the world.
-* Tool (`register_tool`): A tool/weapon that can dig and damage
-  things according to `tool_capabilities`.
-* Craftitem (`register_craftitem`): A miscellaneous item.
+* Node: An item that can be placed in the world as a node.
+* Tool: An item that can be worn out and repaired, cannot be stacked, and often
+  digs or deals damage.
+* Craftitem: A general purpose item that can be used in crafting recipes, eaten,
+  placed, or used on things in the world.
```

Comment:
Not sure about 'placed', maybe 'dropped'?

Reply:
Items can be placed; doors in MTG are craftitems. And everything can be dropped.

Reply:
Special code places door nodes when a door craftitem is 'placed'. Craftitems can't naturally be placed like nodes, only dropped as objects.

Reply:
ok

Reply:
Now fixed.